WebService/v1/Models/Helpers/IotHubConnectionStringManager.cs

```diff
@@ -0,0 +1,189 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.IO;
+using System.Text.RegularExpressions;
+using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices.Common;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Exceptions;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Models.Helpers
+{
+    public class IotHubConnectionStringManager
+    {
+        private const string USE_LOCAL_IOTHUB = "pre-provisioned";
+        private const string CONNSTRING_FILE_PATH = @"keys\user_iothub_key.txt";
```

Comment:
file paths should be cross platform, there is constant you can use for folder separator

Reply:
the path might not exist, perhaps write in the "current" folder instead?